cli/findsecbugs.sh

```diff
@@ -1,4 +1,5 @@
+#!/bin/bash
 
-SOURCE="${BASH_SOURCE[0]}"
+cd $(dirname $(readlink `[[ $OSTYPE == linux* ]] && echo "-f"` $0))
```

Comment:
`cd` will probably not work if a relative path is passed.

`./findsecbugs.sh someapp.jar`

Assuming the CLI folder is set in the **PATH**.